syntax = "proto3";

package proto;

import "request.proto";
import "response.proto";

message CacheEntry {
  string executor = 1;
  Request request = 2;
  Response response = 3;
}

message CacheData {
  repeated CacheEntry entry = 1;
}
